src/MOIWrapper.jl

```diff
@@ -71,266 +52,204 @@ const solveoptionmap = Dict(
+        solveoptionmap[name](optimizer.solveroptions,value)
     else
         error("Unrecognized option: $name")
     end
 end
 
 function ClpOptimizer(;kwargs...)
-    m = ClpOptimizer(nothing)
-    m.params = Dict{String,Any}()
-    MOI.empty!(m)
+    optimizer = ClpOptimizer(nothing)
+    optimizer.params = Dict{String,Any}()
+    MOI.empty!(optimizer)
     for (name,value) in kwargs
-        m.params[Symbol(name)] = value
+        optimizer.params[Symbol(name)] = value
     end
-    return m
+    return optimizer
 end
 
-LQOI.LinearQuadraticModel(::Type{ClpOptimizer},env) = ClpModel() 
+LQOI.LinearQuadraticModel(::Type{ClpOptimizer},env) = ClpModel()
 
-LQOI.supported_constraints(s::ClpOptimizer) = SUPPORTED_CONSTRAINTS
-LQOI.supported_objectives(s::ClpOptimizer) = SUPPORTED_OBJECTIVES
+LQOI.supported_constraints(optimizer::ClpOptimizer) = SUPPORTED_CONSTRAINTS
+LQOI.supported_objectives(optimizer::ClpOptimizer) = SUPPORTED_OBJECTIVES
 
 """
-    change_variable_bounds!(instance, cols::Vector{Int}, 
-        values::Vector{Float64}, senses::Vector)
-        
-Change the bounds of the variable.
+    replace_inf(x::Vector{<:Real})
+
+Replaces any occurances of an element `x[i]>1e20` with `Inf` and `x[i]<-1e20`
+with `-Inf`.
+"""
+function replace_inf(x::Vector{<:Real})
+    for i in 1:length(x)
+        if x[i] > 1e20
+            x[i] = Inf
+        elseif x[i] < -1e20
+            x[i] = -Inf
+        end
+    end
+    return x
+end
+
 """
-function LQOI.change_variable_bounds!(instance::ClpOptimizer, cols::Vector{Int}, 
+    replace_inf(x::Real)
+
+Return `Inf` if `x>1e20`, `-Inf` if `x<-1e20`, and `x` otherwise.
+"""
+function replace_inf(x::Real)
+    if x > 1e20
+        return Inf
+    elseif x < -1e20
+        return -Inf
+    else
+        return x
+    end
+end
+
+function LQOI.change_variable_bounds!(instance::ClpOptimizer, cols::Vector{Int},
         values::Vector{Float64}, senses::Vector)
-    
-    upperbounds = replace_inf(get_col_upper(instance.inner))    
-    lowerbounds = replace_inf(get_col_lower(instance.inner))    
-    
+    upperbounds = replace_inf(get_col_upper(instance.inner))
+    lowerbounds = replace_inf(get_col_lower(instance.inner))
     for i in 1:length(senses)
         if senses[i] == Cchar('U')
             upperbounds[cols[i]] = values[i]
         elseif senses[i] == Cchar('L')
             lowerbounds[cols[i]] = values[i]
-        else            
-            error("sense is " * string(senses[i]) * ", but only " * Cchar('U') *  
-            " and " * Cchar('L') * " senses are supported")
+        else
+            error("sense is $(senses[i]), but only 'U' and 'L' are supported")
         end
     end
-    
     chg_column_upper(instance.inner, upperbounds)
     chg_column_lower(instance.inner, lowerbounds)
 end
 
-"""
-    get_variable_lowerbound(instance, col::Int)::Float64
-
-Get the lower bound of the variable in 1-indexed column `col` of the model `m`.
-"""
```

Comment:
Why not having the comments here? Even if they are in LQOI, it's handy to not go to LQOI each time to check what the function is supposed to do, when developing.

Reply:
If someone goes `? LQOI.get_variable_lowerbound` they will see it twice. We tried to name the methods so that they were pretty clear what was going on. I guess we could have comments instead of docstrings.

Reply:
Is it bad ? if we type instance in the doc strings and add some info, one would see for example:

````julia
  get_variable_lowerbound(instance, col::Int)::Float64

  Get the lower bound of the variable in 1-indexed column col of the model m.
  It is an interface function that is part of LinQuadOptInterface.
  
  get_variable_lowerbound(instance::ClpOptimizer, col::Int)::Float64

  Get the lower bound of the variable in 1-indexed column col of the model m.

  get_variable_lowerbound(instance::GurobiOptimizer, col::Int)::Float64

  Get the lower bound of the variable in 1-indexed column col of the model m.
````

As a user, this will help knowing which solver among those you are playing with has a given functionality.

I don't know if there is a convention concerning this in julia. But I think it's good to have one, and use the same in all MOI packages. @mlubin what do you think?

Reply:
The problem is that these shouldn't be used by the user. They should use MOI. These are strictly LQOI internal functions.